src/pages/stablecoins.js

```diff
@@ -0,0 +1,882 @@
+            {tokens.map((token, idx) => {
+              return (
+                <TokenCard
+                  key={idx}
+                  emoji={token.emoji}
+                  description={token.description}
+                  emojiSize={3}
+                />
+              )
+            })}
+          </LeftColumn>
+          <StyledGhostCard>
+            <Emoji svg size={3} text=":pizza:" />
+            <h3>The infamous Bitcoin pizza</h3>
+            <p>
+              In 2010, someone bought 2 pizzas for 10,000 bitcoin. At the time
+              these were worth ~$41 USD. In today’s market that’s millions of
+              dollars. There are many similar regretful transactions in
+              Ethereum’s history. Stablecoins solve this problem, so you can
+              enjoy your pizza and hod on to your ETH.{" "}
+            </p>
+          </StyledGhostCard>
+        </TwoColumnContent>
+        <DaiBanner>
+          <StyledLeftColumn>
+            <div>
+              <DaiH2>Dai stablecoin</DaiH2>
+              <DaiSubtitle>
+                Dai is probably the most famous decentralized stablecoin. It’s
+                value is roughly a dollar and it’s accepted widely across dapps.{" "}
+              </DaiSubtitle>
+              <DaiButtonRow>
+                <b>Got ETH and a wallet?</b>{" "}
+                <DaiButton isSecondary ml={"1rem"} to="https://1inch.exchange">
+                  Swap for Dai
+                </DaiButton>
+              </DaiButtonRow>
+              <DaiButtonRow>
+                <b>Need help getting started?</b>{" "}
+                <DaiButton isSecondary ml={"1rem"} to="#explore">
+                  How to get Dai
+                </DaiButton>
+              </DaiButtonRow>
+            </div>
+          </StyledLeftColumn>
+          <Hero
+            fluid={data.dailarge.childImageSharp.fluid}
+            alt="The Dai logo"
+          />
+        </DaiBanner>
+      </Content>
+      <Content>
+        <H2 id="explore">How to get stablecoins</H2>
+      </Content>
+      <FullWidthContainer>
+        <StablecoinAccordion />
+      </FullWidthContainer>
+      <Divider />
+      <Content>
+        <StyledCalloutBanner
+          title="Use your stablecoins"
+          description="Check out Ethereum’s dapps – stablecoins are often more useful for everyday transactions."
+          image={data.doge.childImageSharp.fluid}
+          maxImageWidth={600}
+          alt="Illustration of a doge."
+        >
+          <div>
+            <ButtonLink to="/dapps/">Explore dapps</ButtonLink>
+          </div>
+        </StyledCalloutBanner>
+        <TwoColumnContent>
+          <LeftColumn>
+            <H2>Save with stablecoins</H2>
+            <p>
+              Stablecoins often have an above-average interest rate because
+              there’s a lot of demand for borrowing them. There are dapps out
+              there that let you earn interest on your stablecoins in real time.
```

Comment:
Worth comparing to a savings account here? Banks do the same thing with your deposits - they lend them out & give you an interest rate.